docs/project/principles/safety_strategy.md

```diff
@@ -695,7 +695,7 @@ Advantages:
     memory, and disk space).
     -   This will sometimes be significant, and as a result of multiple checks,
         could be the difference between the hardened build mode being 50% slower
-        than the performance build mode and being 200% slower.
+        than the performance build mode instead of being 200% slower.
```

Comment:
I think we should rephrase this, but I think this rephrasing doesn't quite work: we've gone from "difference between X and Y" to "difference between X instead of Y", which doesn't sound grammatical to me.

How about something like "could *result in* the hardened build mode being 50% slower than the performance build mode instead of being 200% slower"?